src/includes/debug-symbols-apple/_default.mdx

```diff
@@ -34,9 +34,9 @@ Once the integration is set up it will ensure that new builds are detected and f
 
 ![A fully set-up App Store Connect Integration](asc-repository.png)
 
-<Alert level="Note">
+<Alert level="info">
```

Comment:
We try to reserve notes at this level for information that's critical to know; things at the same level of versions that introduce breaking changes. And we try to use them infrequently. If you think this meets that standard, then leave as is. If not, then go with level "Note".
```suggestion
<Alert level="Note">
```